app/containers/ConfirmBattleContainer.jsx

```diff
@@ -0,0 +1,53 @@
+import React, {Component} from 'react'
+import ConfirmBattle from '../components/ConfirmBattle'
+import { getPlayersInfo } from '../utils/githubHelpers'
+
+class ConfirmBattleContainer extends Component {
+  constructor (props, context) {
+    super (props, context)
+
+    this.state = {
+      isLoading: true,
+      playersInfo: []
+    }
+    context.router
+
+    this.handleInitiateBattle = this.handleInitiateBattle.bind(this)
+  }
+
+  async componentDidMount () {
+    const { query } = this.props.location
+    try {
+      const players = await getPlayersInfo([query.playerOne, query.playerTwo])
+      this.setState({
+        isLoading: false,
+        playersInfo: [players[0], players[1]]
```

Comment:
players is already an Array, why not just have `playersInfo: players` here?

or set `const playersInfo = await...` and then set state 
```
{ 
   isLoading: false,
   playersInfo
}